release_notes/unreleased.md

```diff
@@ -1 +1,2 @@
 **Unreleased**
+* Initial release
```

Comment:
Initial release with Python3 support

Also, add notes like  "This app is a replacement for the Azure AD app with Microsoft Graph API as Azure AD API is going to deprecate in near future."

Reply:
updated as suggested